src/Monocle/Backend/Queries.hs

```diff
@@ -1278,9 +1277,8 @@ metricChangesMerged = changeEventCount mi EChangeMergedEvent
       "changes_merged"
       "Changes merged count"
       "The count of changes merged"
-      ( Just $
-          "The metric is the count change merged events."
-            <> changeEventFlavorDesc
+      ( "The metric is the count change merged events."
+          <> changeEventFlavorDesc
```

Comment:
We might want to have a look at string interpolation library to make our life easier. Perhaps https://hackage.haskell.org/package/string-interpolate or even https://hackage.haskell.org/package/PyF

Reply:
I've add the use of string-interpolate. +1